go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/cyberark/conjur-authn-k8s-client
 
-go 1.17
+go 1.16
```

Comment:
Downgraded to correlate with the Dockerfile version. Needs further discussion.

Reply:
ok, we can follow up after this PR gets in.